Makefile

```diff
@@ -13,7 +13,7 @@ tests_pre_commit:
 run_tests: tests
 
 tests_only:
-	poetry run pytest --cov=./ --cov-report=xml --cov-report=html -vv
```

Comment:
it was failing on CI, don't know why, need to check it after.

Reply:
Maybe coverage requirements?  As an aside probably can consider moving towards using [poetry scripts](https://python-poetry.org/docs/cli/#run) in the future when time frees up